getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -1,5 +1,7 @@
 .. _doc_exporting_for_android:
 
+**This page covers the basics** for Android export.  For a more comprehensive walkthrough (*using Godot 3.1*), please [see this excelent tutorial](https://godotlearn.com/godot-3-1-how-to-export-for-android/).
```

Comment:
Is this part really necessary? I'd rather improve the docs we currently have rather than linking to many external sites.

Reply:
it's very well written and includes additional info.  but ok not to link  (it's details are beyond the minimum)